### PR TITLE
Feat(code): Fixes the ship info and Hardpoint tooltip system to work again.

### LINKED ISSUE #154: [Bug] Missing tooltips on the Player Info, Ship Info, and Hardpoint Info Panels
### Describe the bug

These three panels should have tooltips that pop up when hovering over various statistics. And they don't. Hovering does work in other areas, such as outfitters and shipyard. Just not in these three panels.

### Steps to Reproduce

1. Go to player info panel
2. Mouse over a statistic
3. See error

### Expected Behavior

Tooltips should show up.

### Screenshots

_No response_

### Link to save file

_No response_

### Operating System

Windows 10 Pro

### Game Source

GitHub Releases

### Game Version

Endless Sky Delta 0.10.10, 0.10.9, 0.10.8

### Additional Information

I'll be spending a bunch of time trying to solve this, but extra eyes that spot likely looking things would be much appreciated.

My suspicion is that it has something to do with the addition of the hardpoint panel, but I haven't actually narrowed it down specifically to that yet.

### PR DESCRIPTION
**Pull request by Zitchas**
This fixes https://github.com/endless-sky/Endless-Sky-Delta/issues/154

Changes:
- Adds new code to properly pull tooltips for various sections of the ship info and hardpoint info panels that were broken out of the monolithic block in https://github.com/endless-sky/Endless-Sky-Delta/pull/46
- Adds new tooltips for the new attributes referred to in some areas.
- Deletes some duplicate tooltip entries.
- Changes the tooltip hover time required to trigger the tooltip down from 60 to 30. (Which I believe is 1 second down to half a second)

Thanks to Warpcore who provide a brief explanation on the bug issue that helped me get started on the right path, and to VitalChip who provided insight and suggestions.

### CODE REVIEW

File: source/ShipInfoDisplay.cpp [hunk 1]
```diff
@@ -615,29 +615,33 @@ void ShipInfoDisplay::DrawShipHealthStats(const Ship &ship, const Rectangle & bo
 
 
 	// This allows the section to stack nicely with other info panel sections,
-	// But will also allow it to be called on its own in a new box if desire.
+	// But will also allow it to be called on its own in a new box if desired.
```
Comment (TheGiraffe3) [inline source/ShipInfoDisplay.cpp]: I mean, while we're changing this line...
```suggestion
	// but will also allow it to be called on its own in a new box if desired.
```

**Review by TheGiraffe3 — APPROVED** [review]
The Ubuntu check does that periodically. Re-running checks with an empty commit should fix that.

**Comment by Zitchas** [comment]
I can also just manually the-run checks.

**Comment by TheGiraffe3** [comment]
No need now though.

**Comment by TheGiraffe3** [comment]
Nice!

**Pull request merged**